homeassistant/auth/mfa_modules/totp.py

```diff
@@ -76,12 +78,13 @@ def input_schema(self) -> vol.Schema:
 
     async def _async_load(self) -> None:
         """Load stored data."""
-        data = await self._user_store.async_load()
+        async with self._init_lock:
```

Comment:
Same comment as above about needing to check that the output of this method hasn't ran (`self._users is not None`)